edx_proctoring/templates/proctoring/seq_proctored_exam_verified.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="success sequence proctored-exam passed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session was reviewed and passed all requirements
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="success"> Passed </b>
```

Comment:
"Your proctoring session results:" 
"Passed" -> "Satisfactory"
(We are going with "Satisfactory" and "Unsatisfactory" right?)